cmake/modules/CheckNasm.cmake

```diff
@@ -0,0 +1,63 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright(c) 2020, Intel Corporation
+#
+# Author: Changcheng Liu <changcheng.liu@aliyun.com>
```

Comment:
i am not sure about these. couple issues

- license: why did you decide to release this source file under Apache-2.0? as the major part of this source file comes from `cmake/modules/CheckYasm.cmake` which was created by me see 78aad3a28cd83f2c3bbb8ad8efeda5ff04f0252f. it's licensed under LGPL 2.1. and i contributed the major part of this macro in e06046c3d9a8dcc03f28b6746d57cc6279d063ee .
- copyright: by adding this line. you claim that Intel has the copyright of this file. but i don't agree. as explained above, i contributed the major part of this creative work when working for Red Hat. so i think Red Hat at least also co-owns the copyright. 
- author: as explained above. i don't think it's appropriate to list only a single contributor here. as this is a collective work.

Reply:
@tchaikov 
I'll clean the header.
BTW, this is added by my vim configuration.

Reply:
@tchaikov 
Done
Clean header in ```cmake/modules/CheckNasm.cmake``` and ```src/nasm-wrapper```